### PR TITLE
Fix: Upgrade vendored dockershim to v1.10.0

### PR DESCRIPTION
**Pull request by liyanyanli**
**1.Describe what this PR did**

Upgrade vendored Kubernetes to 1.10.0

**_PrivilegedRuntime is still just a skeleton. If this skeleton is appropriate, I will continue..._**

**2.Does this pull request fix one issue?**

Derivation of #316 

**3.Describe how you did it**

Upgrade vendored Kubernetes to v1.10.0

**4.Describe how to verify it**
**critool:** 
    version： master branch

```
root@frakti-u-1:~/ferry# ./crictl ps
CONTAINER ID        IMAGE               CREATED             STATE               NAME                ATTEMPT
root@frakti-u-1:~/ferry# ./crictl --runtime-endpoint /var/run/frakti.sock ps
CONTAINER ID        IMAGE               CREATED             STATE               NAME                ATTEMPT

```

**5.Special notes for reviews**
None

### CODE REVIEW

**Comment by resouer** [comment]
Also, cc @bergwolf @liwei

**Comment by liyanyanli** [comment]
**Describe how to verify it**

**Env:**
k8s:1.10.0

**cri-tool:**

NONE

**Kubernetes:**

runtime-docker:

Create pod:
```
root@frakti-u-1:~# kubectl create -f frakti-docke1r.yaml
deployment.apps "frakti-docker-1" created
root@frakti-u-1:~# kubectl create -f frakti-docker.yaml 
deployment.apps "frakti-docker" created
root@frakti-u-1:~# kubectl get pod 
NAME                               READY     STATUS    RESTARTS   AGE
frakti-docker-1-7cfffb9849-sm4q9   1/1       Running   0          38s
frakti-docker-7cfffb9849-zwsbx     1/1       Running   0          12s
```
Cross host network:
```
root@frakti-u-1:~# docker exec -it a9e03fe92861 bash
[ root@frakti-docker-1-7b75d7fff4-vd7zh:/etc/nginx ]$ ping 10.40.0.4
PING 10.40.0.4 (10.40.0.4) 56(84) bytes of data.
64 bytes from 10.40.0.4: icmp_seq=1 ttl=64 time=1.84 ms
64 bytes from 10.40.0.4: icmp_seq=2 ttl=64 time=0.434 ms
^C
--- 10.40.0.4 ping statistics ---
2 packets transmitted, 2 received, 0% packet loss, time 1001ms
rtt min/avg/max/mdev = 0.434/1.137/1.841/0.704 ms
```
Delete deployment:
```
root@frakti-u-1:~# kubectl delete deployment frakti-docker-1
deployment.extensions "frakti-docker-1" deleted
root@frakti-u-1:~# kubectl get pod -owide
NAME                             READY     STATUS    RESTARTS   AGE       IP          NODE
frakti-docker-7cfffb9849-zwsbx   1/1       Running   0          22m       10.40.0.4   frakti-u-2
```

File: cmd/frakti/frakti.go [hunk 1]
```diff
@@ -41,7 +41,7 @@ const (
 )
 
 var (
-	version = pflag.Bool("version", false, "Print version and exit")
+	version = pflag.Bool("versions", false, "Print version and exit")
```
Comment (resouer) [inline cmd/frakti/frakti.go]: why change it to versions?

**Comment by resouer** [comment]
@liyanyanli There's a build error: 
```
pkg/unikernel/metadata/container_test.go:27:2: imported and not used: "k8s.io/frakti/vendor/k8s.io/kubernetes/pkg/kubelet/apis/cri/runtime/v1alpha2"
pkg/unikernel/metadata/container_test.go:33:12: undefined: runtime
pkg/unikernel/metadata/container_test.go:80:13: undefined: runtime
pkg/unikernel/metadata/container_test.go:97:13: undefined: runtime
```

**Comment by liyanyanli** [comment]
Hi, @resouer 
1.The previous verification was already using dockershim(deploy pods to dockershim)
2.Sorry for making such a low level mistake in modifying CRI

File: cmd/frakti/frakti.go (hunk 1 above)
Comment (liyanyanli) [reply cmd/frakti/frakti.go]: Avoid errors: "/usr/bin/frakti flag redefined: version"

Comment (resouer) [reply cmd/frakti/frakti.go]: This is ugly, please remove `version` and print out frakti version whenever it started with:
```
glog.Infof("frakti version: %s", fraktiVersion)
```

Comment (resouer) [reply cmd/frakti/frakti.go]: ping @liyanyanli :)

File: cmd/frakti/frakti.go [hunk 2]
```diff
@@ -41,7 +41,6 @@ const (
 )
 
 var (
-	version = pflag.Bool("version", false, "Print version and exit")
```
Comment (liyanyanli) [inline cmd/frakti/frakti.go]: Removed version and print out frakti version

File: cmd/frakti/frakti.go [hunk 3]
```diff
@@ -71,9 +70,8 @@ func main() {
 	logs.InitLogs()
 	defer logs.FlushLogs()
 
-	if *version {
```
Comment (liyanyanli) [inline cmd/frakti/frakti.go]: Removed version and print out frakti version

File: cmd/frakti/frakti.go (hunk 2 above)
Comment (liyanyanli) [inline cmd/frakti/frakti.go]: Removed the version and print the frakti version at any time.

**Comment by resouer** [comment]
There still other flag error in CI, please fix it. @liyanyanli

**Comment by resouer** [comment]
Some test case failed in CI

**Comment by liyanyanli** [comment]
I will fix it as soon as possible, thanks very much！

**Comment by liyanyanli** [comment]
"reopen container log" is not implemented yet, skip reopening container e2e test temproarily.

**Comment by resouer** [comment]
Gofmt error

**Pull request merged**